pyramid_oereb/standard/load_legend_entries.py

```diff
@@ -102,8 +106,11 @@ def create_legend_entries_in_standard_db(config, topic_code, temp_creation_path=
                     layer_config.get('layer') == params.get('LAYERS')[0]:
                 layer_existent = True
         if not layer_existent:
+            service_url = urlunsplit((url.scheme, url.netloc, '', '', ''))
+            if replace_host is not None:
+                service_url = replace_host
```

Comment:
This can be also be done by:
```python
if replace_host is None:
  service_url = urlunsplit((url.scheme, url.netloc, '', '', ''))
else:
  service_url = replace_host
```
or shorter:
```python
service_url = urlunsplit((url.scheme, url.netloc, '', '', '')) \
   if replace_host is None else replace_host
```
With that we don't calculate `urlunsplit` if not needed :-)